runtime/Java/pom.xml

```diff
@@ -23,10 +23,25 @@
 	<properties>
 		<!-- Assumes dot is in the system path, or specified for the build. -->
 		<dot.path>dot</dot.path>
-	</properties>
+        <junit.version>5.9.0</junit.version>
+    </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
```

Comment:
I disagree. runtime (`runtime/Java`) and tests (`tool-testsuite`, `runtime-testsuite`) modules are currently separated, I don't see a reason why tests should be placed in all these modules. Please move all tests (I guess `TestCodePointCharStream`) to `tool-testsuite` or `runtime-testsuite`).

Reply:
runtime-testsuite is about end-to-end testing i.e. from a grammar and input ensure the output is correct.
it tests the runtime in context

unit-testing a BitSet has nothing to do with end-to-end testing.
unit testing of runtime-specific classes should be with their respective modules.
In antlr4 you'll find that Dart, JS, Python and Swift unit tests are with the runtime.
So this is fixing an inconsistency, not creating one

(I suspect tool-testsuite sits in a dedicated module because it has dependencies on runtime and runtime-testsuite)